docinfo.adoc

```diff
@@ -0,0 +1,36 @@
+== Document information
+
+=== Copyright and license information
+This specification is licensed under the Creative Commons
+Attribution 4.0 International License (CC-BY 4.0). The full
+license text is available at
```

Comment:
I suggest the Apache 2.0 License because it is our current unified license for open source project.

Reply:
The LICENSE file of this repo already contains the Apache 2.0 license.
So this section was not in sync with the contents of the LICENSE file.
I've addressed this issue with the following commit: https://github.com/T-head-Semi/thead-extension-spec/pull/1/commits/e90c8a5731c263ce3a190d161352b57882af8365

Thanks!